monitor_frontend/monitor_frontend.model.lkml

```diff
@@ -3,7 +3,7 @@ label: "Firefox Monitor (Frontend)"
 # Include files from looker-hub or spoke-default below. For example:
 include: "//looker-hub/monitor_frontend/explores/*"
 # include: "//looker-hub/monitor_frontend/dashboards/*"
-# include: "//looker-hub/monitor_frontend/views/*"
-# include: "views/*"
+include: "//looker-hub/monitor_frontend/views/*"
```

Comment:
I'd actually leave this include-all-hub-views line commented out, because the spoke view refinement is already including the specific hub view it's referencing.
```suggestion
# include: "//looker-hub/monitor_frontend/views/*"
```

Reply:
This is still true, but it isn't a big deal, more of a minor optimization.